cmd/opentelemetry/app/internal/reader/es/esdependencyreader/dependency_store.go

```diff
@@ -87,7 +87,7 @@ func (r *DependencyStore) GetDependencies(ctx context.Context, endTs time.Time,
 	searchBody := getSearchBody(endTs, lookback)
 
 	indices := dailyIndices(r.indexPrefix, endTs, lookback)
-	response, err := r.client.Search(ctx, searchBody, defaultDocCount, indices...)
+	response, err := r.client.Search(ctx, searchBody,defaultMaxDocCount, indices...)
```

Comment:
Could we make the `defaultMaxDocCount` configurable here as well?

Reply:
Good catch, I missed this one.